### DISCUSSION TITLE
[GOOGLE-CHAT] Unable to find a way to create a 1:1 conversation

### DISCUSSION THREAD

**Discussion by D0nte53** [googleapis/google-api-nodejs-client#3615] [Q&A]
Hello,

I am looking for a way to send a private message through the Google Chat API. First, I had to sign up for Google Workspace (I'm still not sure why, as it is sometimes mentioned that Google Chat is accessible to everyone). Once that was done, I was able to create spaces, BUT:

- I cannot create a space with multiple users directly (the space is created with one user, and then the others receive invitations via email). I looked into how to automatically accept invitations, but it doesn't seem to work if everyone is not part of the same workspace. (``chat.spaces.create`` and then for each member ``chat.spaces.members.create`` )

- I am unable to send private messages (1:1 conversations) without an existing common space. (``chat.spaces.messages.create`` with ``privateMessageViewer``)

Thank you for your help.
Have a great day !

**Comment by sofisl** [comment]
Hello! Since this is an API question, transferring this over to Discussions Q&A. Thank you!
